interface/main/calendar/modules/PostCalendar/pntemplates/default/views/header.html

```diff
@@ -13,7 +14,8 @@
 <!-- the javascript used for the ajax_* style calendars -->
 <script type="text/javascript" src="[-php-] echo $GLOBALS['webroot'] [-/php-]/library/dialog.js?v=[-php-] echo $GLOBALS['v_js_includes']; [-/php-]"></script>
 <script type="text/javascript" src="[-php-] echo $GLOBALS['webroot'] [-/php-]/library/textformat.js"></script>
-<script type="text/javascript" src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-min-1-7-2/index.js"></script>
+<script type="text/javascript" src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-min-3-1-1/index.js"></script>
+<script type="text/javascript" src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/bootstrap-3-3-4/dist/js/bootstrap.min.js"></script>
```

Comment:
Looks like a job for super smarty function `{headerTemplate}`:
place `{headerTemplate}` above line 4. Drop lines 4-7, 15-18

Reply:
Could not get this to work with this header template. Don't know why and tried several approaches so leaving as is for now.

Reply:
Turns out the calendar smarty was not picking up the custom smarty functions. Got it working here:
https://github.com/openemr/openemr/pull/1354
(I plan to bring it in after a bit more testing)